### PR TITLE
Prim shapes

### PR DESCRIPTION
**Pull request by lidonohu**
Primitive methods for star and rhombicuboctahedron are in primitive.py as well as the winding order algorithm which is separate (not in primitive and not refined using numpy yet).

### CODE REVIEW

File: fury/primitive.py [hunk 1]
```diff
@@ -328,3 +328,163 @@ def _fexp(x, p):
     vertices = np.ascontiguousarray(xyz)
 
     return vertices, sphere_triangles
+
+def primRhombi():
```
Comment (skoudoro) [inline fury/primitive.py]: Can you rename it `prim_rhombicubohectadron` ?

File: fury/primitive.py [hunk 2]
```diff
@@ -328,3 +328,163 @@ def _fexp(x, p):
...
+    '''Return vertices and triangle for rhombicuboctahedron geometry
```
Comment (skoudoro) [inline fury/primitive.py]: - Can you use double quotes instead of single quotes?
- `triangles` plural

File: fury/primitive.py [hunk 3]
```diff
@@ -328,3 +328,163 @@ def _fexp(x, p):
...
+
+    :return:
```
Comment (skoudoro) [inline fury/primitive.py]: Returns instead of :return:

File: fury/primitive.py [hunk 4]
```diff
@@ -328,3 +328,163 @@ def _fexp(x, p):
...
+                             [2, 3, 14],
+                             [3, 14, 15],
+                             [0, 7, 12],
+                             [7, 12, 19],
+                             [3, 15, 16],
+                             [3, 4, 16],
+                             ], dtype='i8')
+    return my_vertices, my_triangles
+
+def primStar(dim=2):
```
Comment (skoudoro) [inline fury/primitive.py]: `prim_star` instead of `primStar`

File: fury/primitive.py [hunk 5]
```diff
@@ -328,3 +328,163 @@ def _fexp(x, p):
...
+    '''Return vertices and triangle for star geometry
```
Comment (skoudoro) [inline fury/primitive.py]: same comment as above

File: fury/primitive.py [hunk 6]
```diff
@@ -328,3 +328,163 @@ def _fexp(x, p):
...
+
+
```
Comment (skoudoro) [inline fury/primitive.py]: You should add `Parameters` title

File: fury/primitive.py [hunk 7]
```diff
@@ -328,3 +328,163 @@ def _fexp(x, p):
...
+    :param dim: default 2
```
Comment (skoudoro) [inline fury/primitive.py]: `dim : int`instead

File: fury/primitive.py [hunk 8]
```diff
@@ -328,3 +328,163 @@ def _fexp(x, p):
...
+        triangles = np.array([[1, 9, 0],
+                          [1, 2, 3],
+                          [3, 4, 5],
+                          [5, 6, 7],
+                          [7, 8, 9],
+                          [1, 9, 3],
+                          [3, 7, 9],
+                          [3, 5, 7]], dtype='i8')
+
+    if(dim == 3):
```
Comment (skoudoro) [inline fury/primitive.py]: `elif` instead of `if`

**Comment by lidonohu** [comment]
Reopening later

**Pull request closed without merging**